person.rb

```diff
@@ -20,6 +20,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ Required ] I think it would be better if your `add_rental` method employed the use of the intermediate class`Rental` class as sepecified in the project requirements. The `person` object should be sent as a parameter to create the new rental achieving the `has-many` association. Please see the example below for more details on how you can achieve this.

## Example code
```ruby
  def add_rental(person, date)
     Rental.new(date, self, person)
  end
```

## Project requirements
![Screenshot from 2023-08-02 23-25-24](https://github.com/Nahid1911/school-library/assets/16392046/56f6e464-fa1e-4b1e-a10f-c64589f10a8c)

Reply:
Hi @Felix45 
Thank you for your comments add detailed instruction, the issue has been resolved.